experimental/src/test/scala/org/testcontainers/testcontainers4s/containers/SingleContainerSuite.scala

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.testcontainers4s.containers
+
+import org.scalatest.FreeSpec
+import org.testcontainers.testcontainers4s.containers.scalatest.TestContainerForEach
+
+class SingleContainerSuite extends FreeSpec with TestContainerForEach {
```

Comment:
@LMnet I played with new API and seems I was able to make both old and new apis work (at least for a single container)

https://github.com/testcontainers/testcontainers-scala/blob/experiment/experimental/src/test/scala/org/testcontainers/testcontainers4s/containers/SingleContainerSuite.scala

What do you think?